### PR TITLE
List appt reminders

### PR DESCRIPTION
**Pull request by asperkin**

### CODE REVIEW

File: athenahealth/appointment_reminders.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+// approximatedate within the given date range
+//
+// GET /v1/{practiceid}/appointments/appointmentreminders
+//
+// https://docs.athenahealth.com/api/sandbox#/Appointments/getPracticeidAppointmentsAppointmentreminders
+func (h *HTTPClient) ListAppointmentReminders(ctx context.Context, opts *ListAppointmentRemindersOptions) (*ListAppointmentRemindersResult, error) {
+	out := &ListAppointmentRemindersResult{}
+
+	q := url.Values{}
+	if len(opts.DepartmentID) > 0 {
```
Comment (asperkin) [inline athenahealth/appointment_reminders.go:54]: Saw this elsewhere, but really should we just early exit in this scenario?

File: athenahealth/appointment_reminders.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+	Pagination *PaginationResult
+}
+
+// ListAppointmentReminders - Retrieves a list of appointment reminders with the specified departmentid and
+// approximatedate within the given date range
+//
+// GET /v1/{practiceid}/appointments/appointmentreminders
+//
+// https://docs.athenahealth.com/api/sandbox#/Appointments/getPracticeidAppointmentsAppointmentreminders
```
Comment (dillonstreator) [inline athenahealth/appointment_reminders.go:49]: can we swap this with `https://docs.athenahealth.com/api/api-ref/appointment-reminders#Get-list-of-appointment-reminders`

File: athenahealth/appointment_reminders.go (hunk 1 above)
Comment (dillonstreator) [reply athenahealth/appointment_reminders.go:54]: yes, we should just early return if DepartmentID is empty

Comment (asperkin) [reply athenahealth/appointment_reminders.go]: done

**Pull request merged**